movies/movies/Extension.swift

```diff
@@ -0,0 +1,45 @@
+// Extension.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// загрузка картинки
+extension UIImageView {
+    func loadImage(with url: String, placeHolder: UIImage? = nil) {
+//        guard url != nil else { return }
+        image = nil
+        let iconUrl = "https://image.tmdb.org/t/p/w500\(url)"
```

Comment:
```suggestion
        let iconUrl = "(url)"
```
это надо было вынести либо константы, либо в сетевой слой в ендпойнты